### PR TITLE
(#839) Time Picker Pass Though

### LINKED ISSUE #839: Time Picker USWDS Pass Through to NCIDS
### USWDS Component Name
[Time Picker](https://designsystem.digital.gov/components/time-picker/)

### USWDS Component Packages
* Package usage: `@forward "usa-time-picker";`
* Dependencies: `uswds-fonts`, `usa-label`, `usa-hint`, `usa-input`

### Subtasks
- [x] Make a folder with the component name under uswds-native
- [x] Copy USWDS twig and any JSON
- [x] Make an index.scss for the stories
- [x] Copy and hack up USWDS stories.js file to use our story format
- [x] Create scenarios.js by adding a scenario for each story in stories.js
- [x] Add link to the USWDS documentation on the NCIDS documentation site

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #839

Add Time Picker pass through component.

### CODE REVIEW

File: docs/content/index.mdx [hunk 1]
```diff
@@ -11,19 +11,19 @@ title: NCI Design System Home
...
 - [Button](./components/button)
 - [CTA Strip](./components/cta-strip)
+- [Feature Card](./components/feature-card)
 - [Footer](./components/footer)
 - [Header](./components/header)
 - [Guide Card](./components/guide-card-row)
 - [Guide Card with Image and Description](./components/guide-card-with-image-and-description)
 - [Icons](./components/icons)
 - [List](./components/list)
+- [NCI Hero](./components/nci-hero)
```
[the remark this answers is not in the document]
Comment (olitharp-nci) [reply docs/content/index.mdx:21]: can you add to your commit message that you fixed the alphabetizing of the components

File: docs/content/index.mdx [hunk 2]
```diff
@@ -47,11 +47,13 @@ title: NCI Design System Home
 - [Radio buttons](https://designsystem.digital.gov/components/radio-buttons/)
 - [Select](https://designsystem.digital.gov/components/select/)
 - [Step indicator](https://designsystem.digital.gov/components/step-indicator/)
+- [Summary Box](https://designsystem.digital.gov/components/summary-box/)
 - [Tag](https://designsystem.digital.gov/components/tag/)
 - [Table](https://designsystem.digital.gov/components/table/)
-- [Summary Box](https://designsystem.digital.gov/components/summary-box/)
+- [Time Picker](https://designsystem.digital.gov/components/ime-picker)
```
Comment (olitharp-nci) [inline docs/content/index.mdx:53]:
```suggestion
- [Time Picker](https://designsystem.digital.gov/components/time-picker)
```

File: docs/content/index.mdx [hunk 3]
```diff
@@ -47,11 +47,13 @@ title: NCI Design System Home
...
 - [Tooltip](https://designsystem.digital.gov/components/tooltip/)
 
+
```
Comment (olitharp-nci) [inline docs/content/index.mdx:56]: fix this extra line

File: docs/content/index.mdx [hunk 4]
```diff
@@ -56,8 +56,13 @@ title: NCI Design System Home
 - [Select](https://designsystem.digital.gov/components/select/)
 - [Step indicator](https://designsystem.digital.gov/components/step-indicator/)
 - [Summary Box](https://designsystem.digital.gov/components/summary-box/)
+  <<<<<<< HEAD
```
Comment (olitharp-nci) [inline docs/content/index.mdx:59]: can you fix this conflict

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

File: testing/ncids-css-testing/.backstop/reference/ncids_usa-in-page-navigation_link_active_0_document_0_tablet.png
Comment (olitharp-nci) [inline testing/ncids-css-testing/.backstop/reference/ncids_usa-in-page-navigation_link_active_0_document_0_tablet.png:1]: 😭 i thought we fixed this

cannot merge this with this regression

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Review by olitharp-nci — APPROVED** [review]

**Review by sarinapadilla — APPROVED** [review]

**Pull request merged**